.github/actions/e2e/install-karpenter/action.yaml

```diff
@@ -47,31 +47,36 @@ runs:
       logout: true
   - name: install-karpenter
     shell: bash
+    env:
+      ECR_ACCOUNT_ID: ${{ inputs.ecr_account_id }}
+      ECR_REGION: ${{ inputs.ecr_region }}
+      ACCOUNT_ID: ${{ inputs.account_id }}
+      CLUSTER_NAME: ${{ inputs.cluster_name }}
+      K8S_VERSION: ${{ inputs.k8s_version }}
     run: |
-      aws eks update-kubeconfig --name "${{ inputs.cluster_name }}"
+      aws eks update-kubeconfig --name "$CLUSTER_NAME"
 
       # Parse minor version to determine whether to enable the webhooks
-      VERSION=${{ inputs.k8s_version }}
-      RELEASE_VERSION_MINOR="${VERSION#*.}"
+      RELEASE_VERSION_MINOR="${K8S_VERSION#*.}"
```

Comment:
Can you point me to documentation on what this does?

Reply:
Discussed offline. This parses out the RELEASE_VERSION_MINOR by using suffix logic to get ride of anything before the first "dot".